ports/k32l2/boards/frdm-k32l2b/board.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Ha Thach for Adafruit Industries
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef BOARD_H_
+#define BOARD_H_
+
+//--------------------------------------------------------------------+
+// Flash
+//--------------------------------------------------------------------+
+
+#define BOARD_FLASH_SIZE      0x40000U
+
+//--------------------------------------------------------------------+
+// Button
+//--------------------------------------------------------------------+
+
+#define BUTTON_GPIO           GPIOC
+#define BUTTON_PORT           PORTC
+#define BUTTON_CLK_PORT       kCLOCK_PortC
+#define BUTTON_PIN            3
+#define BUTTON_STATE_ACTIVE   0
+
+//--------------------------------------------------------------------+
+// LED
+//--------------------------------------------------------------------+
+
+#define LED_GPIO              GPIOD
+#define LED_PORT              PORTD
+#define LED_CLK_PORT          kCLOCK_PortD
+#define LED_PIN               5
+#define LED_STATE_ON          0
+
+//--------------------------------------------------------------------+
+// Neopixel
+//--------------------------------------------------------------------+
+
+// Number of neopixels
+#define NEOPIXEL_NUMBER       0
+
+//--------------------------------------------------------------------+
+// USB UF2
+//--------------------------------------------------------------------+
+
+// 0x1FC9, 0x0154 is reserved for TinyUF2 usage
+#define USB_VID           0x1FC9
+#define USB_PID           0x0154
```

Comment:
does NXP explicitly allocate this PID for TinyUF2 usage ?

Reply:
Yes, I requested a dedicated PID for TinyUF2 usage.
The plan is to use it for TinyUF2 on any NXP based board.

Reply:
thanks for your confirmation. Preferably each product/board will have its own PID, though sharing the PID for NXP MCU sound reasonable as well. Did NXP approve your request yet ?

Reply:
> Yes, that is the numbers they signed for this.
> I can dm the message if that helps.
> Thanks,
> Greg

thanks, this is good enough.